docs/markers.md

```diff
@@ -124,3 +124,13 @@ A second marker type `+operator-builder:collection:field` can be used with the
 same arguments as a Field Marker. These markers are used to define global Fields
 for your Collection and can be used in any of its associated components.
 
+Important: If you use a marker on a [collection
+resource](workload-collections.md#collection-resources) do *not* use the
```

Comment:
@lander2k2 should we add validation to the marker parsing to spit back a friendly error message? Or can we mutate the field into a collection field when we find a collection marker on a collection resource?  I think we may want to do the latter so we can avoid errors and bad UX when these markers are present.

Reply:
Yes, I agree.  I'll look into your second suggestion.  Failing that, I'll see if I can add validation.

Reply:
@scottd018  Pushed a commit that treats all markers on collection resources as collection markers.

Reply:
@lander2k2 boom!  LGTM!